consensus/istanbul/core/message_set.go

```diff
@@ -36,13 +36,26 @@ func newMessageSet(valSet istanbul.ValidatorSet) *messageSet {
 
 // ----------------------------------------------------------------------------
 
-type messageSet struct {
+type MessageSet interface {
+	fmt.Stringer
+	Add(msg *istanbul.Message) error
+	GetAddressIndex(addr common.Address) (uint64, error)
+	GetAddressPublicKey(addr common.Address) ([]byte, error)
+	ValSetSize() uint64
+	Remove(address common.Address)
+	Values() (result []*istanbul.Message)
+	Size() int
+	Get(addr common.Address) *istanbul.Message
+	// String() string
```

Comment:
Why is this commented out?

Reply:
oh, that's the Stringer interface 
i'll remove the comment